src/components/NavBar/NavBar.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<transition name = "navbar_fade">
-		<div class="navbar" v-show="visible">
+		<div class="navbar" v-show="visible">		
```

Comment:
Additional whitespace should be removed. I assume it may be because your text editor is not configured to use .editorconfig files.